src/app/features/ledger/generic-steps/connect-device/connect-ledger-start.tsx

```diff
@@ -43,8 +49,8 @@ export function ConnectLedgerStart() {
   return (
     <Sheet isShowing header={<SheetHeader />} onClose={() => navigate('../')}>
       <ConnectLedger
-        connectBitcoin={() => connectChain('bitcoin')}
-        connectStacks={() => connectChain('stacks')}
+        connectBitcoin={!hasBitcoinLedgerKeys ? () => connectChain('bitcoin') : undefined}
+        connectStacks={!hasStacksLedgerKeys ? () => connectChain('stacks') : undefined}
```

Comment:
Not sure I understand this part?

Reply:
This was just a way to make sure this component only shows the button to connect each chain, it defaults to showing both, but I'm a little unclear why this component was created? It looks like it is handling unsupported browsers? I can also just create a diff component and pass in the chain from the gate.

Reply:
I removed this so will resolve. I ended up creating separate components rather than reusing this one that is for onboarding.